gallery/src/components/NxStatefulDropdownIconOnly/NxStatefulDropdownIconOnlyPage.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxTable, NxCode, NxP, NxTextLink, NxFontAwesomeIcon } from '@sonatype/react-shared-components';
+import { faEllipsisV } from '@fortawesome/free-solid-svg-icons';
+
+import {GalleryDescriptionTile, GalleryExampleTile} from '../../gallery-components/GalleryTiles';
+
+import NxStatefulDropdownIconOnlyExample from './NxStatefulDropdownIconOnlyExample';
+
+const nxStatefulDropdownIconOnlyExampleCode = require('./NxStatefulDropdownIconOnlyExample?raw');
+
+const NxStatefulDropdownPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <NxP>
+        A stateful variation of the dropdown menu component which uses a single icon as its toggle.
+        <NxCode>NxStatefulDropdownIconOnly</NxCode> shares all of the same props as{' '}
+        <NxTextLink href="#/pages/NxStatefulDropdown">NxStatefulDropdown</NxTextLink>{' '}
+        as well as the ones listed below.
+      </NxP>
+      <NxTable>
+        <NxTable.Head>
+          <NxTable.Row>
+            <NxTable.Cell>Prop</NxTable.Cell>
+            <NxTable.Cell>Type</NxTable.Cell>
+            <NxTable.Cell>Required</NxTable.Cell>
+            <NxTable.Cell>Details</NxTable.Cell>
+          </NxTable.Row>
+        </NxTable.Head>
+        <NxTable.Body>
+          <NxTable.Row>
+            <NxTable.Cell>icon</NxTable.Cell>
+            <NxTable.Cell>string</NxTable.Cell>
+            <NxTable.Cell>Yes</NxTable.Cell>
+            <NxTable.Cell>
+              The name of the FontAwesome icon to use in the trigger. The default is <NxCode>faEllipsisV</NxCode>
```

Comment:
Not the name, the object definition

Reply:
> Not the name, the object definition

https://github.com/sonatype/sonatype-react-shared-components/pull/465/commits/94abc6ae861cbd13147bbbf25d88876f640c708b